arm/Microsoft.Network/privateDnsZones/.parameters/parameters.json

```diff
@@ -15,6 +15,19 @@
                 }
             ]
         },
+        "a": {
```

Comment:
I'd suggest to deploy at least one instance for each child. Otherwise they are never tested